profiles/nuvla.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "Nuvla",
+  "revision": "2015-10-13",
+  "vendor_verified": "2015-10-13",
+  "url": "http://nuv.la/login",
```

Comment:
Indeed, but `http://nuv.la` responds with a `301 Moved Permanently` followed by a `401 Unauthorized`. This is expected behaviour on the browser (you get redirected to the login screen, if not logged in) but raises a warning during `rake test`:

```
WARN: Nuvla url must respond with status OK
```

What do you prefer here?